config/config.go

```diff
@@ -60,6 +60,9 @@ const (
 	SettingGateway        = "mender-gateway"
 	SettingGatewayDefault = "localhost:9080"
 
+	SettingWorkflows        = "mender-workflows"
+	SettingWorkflowsDefault = "http://mender-workflows-server:8080"
```

Comment:
Why not just `http://mender-workflows:8080`?

Reply:
@alfrunes because in integration (and in the helm chart) we call it `mender-workflows-server`; I'll think a bit more about it, maybe you are right and we should rename it there as well.

Reply:
@alfrunes thinking about it: it's just a (default) name, let's stick with it. Changing it now requires some work in integration and in the helm chart and brings no value to our users. Both in docker-compose and kubernetes the name of the container / POD is `workflows-server` to differentiate from the `workflows-worker` we'll have later on, thus the default service name makes sense there.